### ISSUE TITLE
clean task is too aggressive

### ISSUE THREAD

**Issue by CREEATION** [haydenbleasel/catalyst#174]
Right now I'm porting my webseite to use Catalyst.
When running `gulp preview`, waiting for all tasks to finish and then editing + saving. a `.js` file for example, **all** tasks are run again, which takes really long and slows down development speed.

**Comment by haydenbleasel** [comment]
@CREEATION Yeah I recently started using Catalyst to power a new project and realised I overlooked a couple of things. I'll update the watch task to be more specific, and only use clean once (or maybe on production only).
